components/reset_status_modal/reset_status_modal.jsx

```diff
@@ -6,8 +6,10 @@ import React from 'react';
 import {FormattedMessage} from 'react-intl';
 import {Preferences} from 'mattermost-redux/constants';
 
-import {toTitleCase} from 'utils/utils.jsx';
+import ModalStore from 'stores/modal_store.jsx';
```

Comment:
We are moving away from using `ModalStore`. We have redux action views in [opening/closing a modal](https://github.com/mattermost/mattermost-webapp/blob/master/actions/views/modals.js). Please search for `ModalIdentifiers` to see how it's implemented.

Reply:
@saturninoabril is there documentation for this new solution?
I may have missed it

Reply:
@miguelespinoza Sorry for late response (as I was out earlier). Though not specifically mentioned, it's part of our campaign in [migrating webapp to redux](https://docs.mattermost.com/developer/webapp-to-redux.html).